engine/gamesys/src/gamesys/components/comp_collision_object.h

```diff
@@ -68,6 +68,7 @@ namespace dmGameSystem
     bool IsCollision2D(void* _world);
     void SetCollisionFlipH(void* _component, bool flip);
     void SetCollisionFlipV(void* _component, bool flip);
+    void Wakeup(void* _world, void* _component);
```

Comment:
Since the namespace is pretty wide, I'd like to have something "collision" related in the name, e.g. "WakeupCollision()"

Reply:
Fixed.